modules/imgproc/src/accum.simd.hpp

```diff
@@ -0,0 +1,2740 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
```

Comment:
Thanks for your suggestion.
I think I'm almost there.  I hope I can find more time tomorrow

Best.